### ISSUE TITLE
Legacy Gitbook link is dead

### ISSUE THREAD

**Issue by saurabhchardereal** [Tencent/rapidjson#1884]
The legacy gitbook link (and domain for that matter) had been deprecated a long time ago. It seems like RapidJson _still_ hasn't migrated to a newer gitbook platform. 

It would be great if there's a gitbook version of the documentation (if not, you could remove the redirect from _readme_)

**Comment by miloyip** [comment]
I tried to migrate but I found that the PDF generation functionality is not free of charge anymore.
I am considering to remove the link.
Any opinion on this?

**Comment by saurabhchardereal** [comment]
Yeah. Just found out that the PDF export feature is now available to only business and enterprise users.

I think you should remove the gitbook link since it serves no purpose now. RapidJson still has Doxygen generated docs which are more than enough (and just in case if anyone really wants Markdown flavored docs they are inside the source anyway).
